src/node/BrowserFetcher.ts

```diff
@@ -497,6 +502,25 @@ function parseFolderPath(
   return {product, platform, revision};
 }
 
+/**
+ * Windows 11 is identified by 10.0.22000 or greater
+ * @internal
+ */
+function _isWindows11(version: string): boolean {
```

Comment:
Let's add a test to make sure it works as expected? I wonder also how trust-worthy `os.release()` is? does Windows use the same format for various versions?

Reply:
Will add a test in a follow-up.